web-console/src/views/datasource-view/datasource-view.tsx

```diff
@@ -425,7 +425,7 @@ GROUP BY 1`;
     return (
       <AsyncActionDialog
         action={async () => {
-          const resp = await axios.delete(
+          const resp = await Api.delete(
             `/druid/coordinator/v1/datasources/${datasourceToMarkAsUnusedAllSegmentsIn}`,
```

Comment:
In the theme of listing all the API endpoints in once file, I think this one could be extracted also for consistency. Maybe extract it as `/druid/coordinator/v1/datasources/%s` and then do a `.replace('%s', datasourceToMarkAsUnusedAllSegmentsIn)` in this function?

Reply:
How about this?
```
`${API_ENDPOINTS.datasources}/${datasourceToMarkAsUnusedAllSegmentsIn}`,
```

Reply:
Done for all the remaining ones.